tests/test_init.py

```diff
@@ -0,0 +1,96 @@
+import unittest
```

Comment:
All unit tests should be located under  eg: "tendrl/commons/tests/test_init.py"

for another example say (tendrl/commons/manager/__init__.py) the tests should be at tendrl/commons/tests/manager/test_init.py